src/Main.java

```diff
@@ -1,5 +1,99 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет, Мир!");
+        System.out.println("Task 1");
+        int age = 18;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен 18, то он совершеннолетний");
+        }
+        int age2 = 17;
+        if (age2 >= 18) {
+            System.out.println("Если возраст человека равен 18, то он совершеннолетний");
+        } else {
+            System.out.println("Если возраст человека равен 17, то он ещё не достиг совершеннолетия, нужно немного подождать");
+        }
+
+        System.out.println("Task 2");
+        int temp = 4;
+        if (temp < 5) {
+            System.out.println("На улице 4 градуса, нужно надеть шапку");
+        }
+        int temp2 = 10;
+        if (temp2 > 5) {
+            System.out.println("На улице 10 градусов, можно идти без шапки");
+        }
+
+        System.out.println("Task 3");
+        int speed = 80;
+        if (speed > 60) {
+            System.out.println("Если скорость " + speed + " км/ч, то водителю придется заплатить штраф");
+        }
+        int speed2 = 50;
+        if (speed2 < 60) {
+            System.out.println("Если скорость " + speed2 + " км/ч, то водителю можно ехать спокойно");
+        }
+
+        System.out.println("Task 4");
+        int ageFrom2To6Years = 2;
```

Comment:
зачем снова новая переменная?
```suggestion
        age = 2;
```